Lib/slapdtest/_slapdtest.py

```diff
@@ -125,7 +125,7 @@ def _add_sbin(path):
 
 def combined_logger(
         log_name,
-        log_level=logging.WARN,
+        log_level,
```

Comment:
Please keep the default value, so that the API doesn't change for other callers.